content/en/news/releases/1.5.x/announcing-1.5/change-notes/index.md

```diff
@@ -4,4 +4,73 @@ description: Istio 1.5 release notes.
+- **Improved** performance of Envoy proxy configuration updates via xDS by sending partial updates where possible [#18354](https://github.com/istio/istio/pull/18354).
+- **Added** an option to configure locality load balancing settings for each targeted service via destination rule[#18406](https://github.com/istio/istio/pull/18406).
+- **Fixed** an issue where pods crashing would trigger excessive Envoy proxy configuration pushes [#18574](https://github.com/istio/istio/pull/18574).
+- **Fixed** issues with applications such as headless services to call themselves directly without going through Envoy proxy [#19308](https://github.com/istio/istio/pull/19308).
+- **Added** detection of `iptables` failure when using Istio CNI [#19534](https://github.com/istio/istio/pull/19534)
+- **Added** `consecutiveGatewayErrors` and `consecutive5xxErrors` as outlier detection options within destination rule [#19771](https://github.com/istio/istio/pull/19771).
+- **Improved** `EnvoyFilter` matching performance [#19786](https://github.com/istio/istio/pull/19786)
+- **Added** support for `HTTP_PROXY` protocol [#19919](https://github.com/istio/istio/pull/19919).
+- **Improved** `iptables` setup to use `iptables-restore` by default [#18847](https://github.com/istio/istio/pull/18847).
+- **Improved** Gateway performance by filtering unused clusters. This setting is disabled by default [#20124](https://github.com/istio/istio/pull/20124).
+
+## Security
+- **Graduated** SDS to stable and enabled by default. It provides identity provisioning for Istio Envoy proxies.
+- **Added** Beta authentication API. The new API separates peer (i.e mutual TLS) and origin (JWT) authentication into [`PeerAuthentication`](https://github.com/istio/api/blob/master/security/v1beta1/peer_authentication.proto) and [`RequestAuthentication`](https://github.com/istio/api/blob/master/security/v1beta1/request_authentication.proto) respectively. Both new APIs are workload-oriented, as opposed to service-oriented in alpha `AuthenticationPolicy`.
+- **Added** [deny semantics](https://github.com/istio/api/blob/master/security/v1beta1/authorization.proto#L28) to Authorization Policy
+- **Graduated** [auto mutual TLS](/docs/tasks/security/authentication/auto-mtls/) from alpha to beta. This feature is now enabled by default.
+- **Improved** node level citadel agent by merging with pod level Istio agent. This improves the security posture by removing the requirement of a pod security policy.
+- **Improved** Istio by including certificate provisioning functionality within istiod.
+- **Improved** Support Kubernetes [`first-party-jwt`](https://kubernetes.io/docs/reference/access-authn-authz/authentication/#service-account-tokens) as a fallback token for CSR authentication in clusters that [`third-party-jwt`](https://kubernetes.io/docs/tasks/configure-pod-container/configure-service-account/#service-account-token-volume-projection) is not supported.
```

Comment:
@fpesce Can you change this item as follows?
"\*\*Added\*\* Support Kubernetes [`first-party-jwt`](https://kubernetes.io/docs/reference/access-authn-authz/authentication/#service-account-tokens) as a fallback token for CSR authentication in clusters where [`third-party-jwt`](https://kubernetes.io/docs/tasks/configure-pod-container/configure-service-account/#service-account-token-volume-projection) is not supported."

Reply:
```suggestion
- **Added** Support Kubernetes [`first-party-jwt`](https://kubernetes.io/docs/reference/access-authn-authz/authentication/#service-account-tokens) as a fallback token for CSR authentication in clusters where [`third-party-jwt`](https://kubernetes.io/docs/tasks/configure-pod-container/configure-service-account/#service-account-token-volume-projection) is not supported.
```